Dockerfile.dapper

```diff
@@ -2,7 +2,7 @@ ARG BASE_BRANCH
 FROM quay.io/submariner/shipyard-dapper-base:${BASE_BRANCH}
 
 ARG PROJECT
-ENV DAPPER_ENV="QUAY_USERNAME QUAY_PASSWORD CLUSTERS_ARGS DEPLOY_ARGS CLEANUP_ARGS E2E_ARGS RELEASE_ARGS MAKEFLAGS FOCUS SKIP PLUGIN E2E_TESTDIR" \
+ENV DAPPER_ENV="QUAY_USERNAME QUAY_PASSWORD CLUSTERS_ARGS DEPLOY_ARGS CLEANUP_ARGS E2E_ARGS RELEASE_ARGS MAKEFLAGS FOCUS SKIP PLUGIN E2E_TESTDIR VERSION CHANNEL" \
```

Comment:
Why is there a need to pass the version? It is calculated by the `make` command

Reply:
The calculated version does not always follow the semver format.
This is how I intend to use it:
https://github.com/submariner-io/submariner-operator/pull/1476/commits/4b1ff16a91cf60f39e37d51bc57eb7e3b050e90c#diff-0a7959ee618128cab3d15fc4fbe17c29729ca35ebd6cbbf6596bf8753a6ad56fR41-R45

Reply:
The calculated version is not intended to be semver, it's for testing and/or publishing dev images and doesnt have semver semantics.
If you want semver you should calculate it yourself.

Anyhow, in your case no need to pass any new env vars anyhow, you can call it like this `make e2e using=olm VERSION=${{ env.VERSION }} FROM_VERSION=${{ env.FROM_VERSION }} CHANNEL=${{ env.CHANNEL }}`

Make makes sure to pass them to the command properly (you can try locally and see).

Reply:
yes I'm already doing so please see this [part](https://github.com/submariner-io/submariner-operator/commit/4b1ff16a91cf60f39e37d51bc57eb7e3b050e90c#diff-0a7959ee618128cab3d15fc4fbe17c29729ca35ebd6cbbf6596bf8753a6ad56fR41-R45)
Unfortunately it does not work without updating the `DAPPER_ENV`.
Just in case you missed, the `Dockerfile.dapper` does not exist on the submariner-operator repo anymore. It is pulled directly from shipyard.

Reply:
@mkolesnik it works :+1:  I used these variables as arguments of `make` instead of environment variables.